feature_server/config/scripts/arena.py

```diff
@@ -36,29 +44,40 @@
 from twisted.internet import reactor
 from twisted.internet.task import LoopingCall
 from piqueserver.commands import add, admin
+import random
+import math
 
 # If ALWAYS_ENABLED is False, then the 'arena' key must be set to True in
 # the 'extensions' dictionary in the map metadata
 ALWAYS_ENABLED = True
 
 # How long should be spent between rounds in arena (seconds)
-SPAWN_ZONE_TIME = 20.0
+SPAWN_ZONE_TIME = 15.0
```

Comment:
For the sake information, if anyone is against chaning these default values, please speak up. There are more changed below, but I'm only adding a comment to this one.